frontend/package.json

```diff
@@ -71,6 +71,10 @@
       "yarn run lint:fix",
       "prettier --write"
     ],
+    "**/*.{ts,tsx}": [
+      "yarn run lint:fix",
+      "prettier --write"
+    ],
```

Comment:
Do we need both this and the one above? Pretty sure the intention is the same with both, but the paths are slightly dirfferent

Reply:
Most likely not, going to find a cleaner solution to this.